com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeBakingProcessSettingsDrawer.cs

```diff
@@ -1,9 +1,10 @@
 using UnityEditor;
+using UnityEditor.Rendering;
 
 namespace UnityEngine.Rendering
 {
     [CustomPropertyDrawer(typeof(ProbeVolumeBakingProcessSettings))]
-    class ProbeVolumeBakingProcessSettingsDrawer : PropertyDrawer
+    internal class ProbeVolumeBakingProcessSettingsDrawer : PropertyDrawer
```

Comment:
Why changing to internal?

Reply:
hum it's a leftover, but internal is the default accessibility for a class anyway